### PR TITLE
chore(dist): optimize size

### PR DESCRIPTION
**Pull request by pladaria**
I was expecting greater gains, but it's something 🤷

---

Analyzing bundles of `page-example-flow-state` in webapp

### Before 

* stat size: 39.66kB => original sources as consumed by webpack
* parsed size: 16.21kB => minified build result
* zip: 5.63kB

### After

* stat size: 38.83kB
* parsed size: 14.52kB
* zip: 4.9kB

### CODE REVIEW

File: src/analytics.ts [hunk 1]
```diff
@@ -146,7 +146,9 @@ export const sanitizeAnalyticsParams = (params: {
     [key: string]: FirebaseValue;
 }): {[key: string]: FirebaseValue} => {
     const sanitizedParams: {[key: string]: FirebaseValue} = {};
-    Object.entries(params).forEach(([key, value]) => {
+    Object.entries(params).forEach((entry) => {
+        const key = entry[0];
+        const value = entry[1];
```
Comment (pladaria) [inline src/analytics.ts:151]: this avoids introducing a polyfill

File: src/device.ts [hunk 2]
```diff
@@ -5,18 +5,18 @@ const TIMEOUT = 200;
...
     postMessageToNativeApp({type: 'IMSI'}, TIMEOUT)
-        .then(({imsi}) => imsi)
+        .then((response) => response.imsi)
         .catch(() => null);
 
 export const requestDeviceImei = (): Promise<string | null> =>
     postMessageToNativeApp({type: 'IMEI'}, TIMEOUT)
-        .then(({imei}) => imei)
+        .then((response) => response.imei)
         .catch(() => null);
```
Comment (pladaria) [inline src/device.ts:20]: micro-optimization. Produces a slightly smaller output

File: src/utils.ts [hunk 3]
```diff
@@ -216,21 +216,14 @@ export const fetch = ({
     headers: {[key: string]: string};
     body: string;
 }): Promise<NativeAppResponsePayload<'FETCH'>> => {
-    if (isWebViewBridgeAvailable()) {
```
Comment (pladaria) [inline src/utils.ts:219]: this wrapping is not needed. The postMessage call fails with the same status if the bridge is not available

**Review by marcoskolodny — APPROVED** [review]

**Review by atabel — APPROVED** [review]
Nice, but I think some of these changes are difficult to maintain. We can remember to avoid async/await, but not so easy to stop using object or array destructuring

**Comment by pladaria** [comment]
> Nice, but I think some of these changes are difficult to maintain. We can remember to avoid async/await, but not so easy to stop using object or array destructuring

yep, it would be nice to get a report of the generated size in PRs

**Pull request merged**